deploy/003_keep3r_job.ts

```diff
@@ -17,15 +18,15 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
       return;
   }
 
-  const companion = await hre.deployments.get('DCAHubCompanion');
+  const companion = await hre.deployments.getOrNull('DCAHubCompanion');
```

Comment:
I have a question about how this would work. If we do this, when we deploy the companion to mainnet, will the keep3r job automatically be re-deployed?